.github/workflows/ci_build_android.yml

```diff
@@ -16,6 +16,8 @@ on:
 jobs:
   build-android-libs:
     runs-on: ubuntu-20.04
+    container:
+      image: xaynetci/yellow:latest
```

Comment:
❔ after we merged https://github.com/xaynetwork/xayn_docker/pull/3 the tag will be the coomit hash of xayn_docker main right?